fns-hl7-pipeline/fn-lake-segs-transformer/src/main/kotlin/gov/cdc/dex/hl7/Function.kt

```diff
@@ -65,77 +65,96 @@ class Function {
+                        outEventList.add(processed_metadata)
+                        logger.info("DEX::Processed OK for Lake of Segments messageUUID: $messageUUID, filePath: $filePath, ehDestination: ${fnConfig.eventHubSendOkName}")
+
+
+                        processedMsgs.add(inputEvent)
+                    } catch (e: Exception) {
+
+                        logger.error("DEX::Exception: Unable to process Message messageUUID: $messageUUID, filePath: $filePath, due to exception: ${e.message}")
+
+
+                        processed_metadata = updateMetadata(
+                            startTime,
+                            PROCESS_STATUS_EXCEPTION,
+                            null,
+                            eventHubMD[messageIndex],
+                            inputEvent,
+                            e,
+                            config
+                        )
+                        context.logger.info("PROCESSED_METADATA = " + processed_metadata)
+                        outEventList.add(processed_metadata)
+                        outErrList.add(gsonWithNullsOn.toJson(processed_metadata))
+                        logger.info("Processed ERROR for Lake of Segments Model messageUUID: $messageUUID, filePath: $filePath, ehDestination: ${fnConfig.eventHubSendErrsName}")
+
+                        processedMsgs.add(inputEvent)
+                    } // .catch
 
-                    // updateMetadata
-                    processed_metadata = updateMetadata(startTime, PROCESS_STATUS_OK, lakeSegsModel, eventHubMD[messageIndex],inputEvent, null, config
-                    )
-                    // add payload to eventhub outbindings for lakeSegsOk
-                    outOkList.add(gsonWithNullsOn.toJson(processed_metadata))
-                    outEventList.add(processed_metadata)
-                    logger.info("DEX::Processed OK for Lake of Segments messageUUID: $messageUUID, filePath: $filePath, ehDestination: ${fnConfig.eventHubSendOkName}")
-
-
-                    processedMsgs.add( inputEvent )
                 } catch (e: Exception) {
 
-                    logger.error("DEX::Exception: Unable to process Message messageUUID: $messageUUID, filePath: $filePath, due to exception: ${e.message}")
-
-
-                    processed_metadata = updateMetadata(startTime, PROCESS_STATUS_EXCEPTION, null, eventHubMD[messageIndex], inputEvent, e, config)
-                    context.logger.info("PROCESSED_METADATA = "+ processed_metadata)
-                    outEventList.add(processed_metadata)
-                    outErrList.add(gsonWithNullsOn.toJson(processed_metadata))
-                    logger.info("Processed ERROR for Lake of Segments Model messageUUID: $messageUUID, filePath: $filePath, ehDestination: ${fnConfig.eventHubSendErrsName}")
-
-                    processedMsgs.add( inputEvent )
+                    // message is bad, can't extract fields based on schema expected
```

Comment:
WE received a message here somehow - so if we're getting an exception - we need to send this message to Err Queue so that we can track it and not lose it - it prob would never happen because a bad message won't make it this far, but for unknown errors - we should log those on err queue w/ summary and problem.
(this was the same issue on Emil's PR for HL7-json. you can talk with him on how to best handle this)